engine/src/main/resources/templates/Advisor/dailyEmailBody.html

```diff
@@ -8,8 +8,9 @@
 <!-- Cards/metrics section -->
 <tr>
   <td>
-    <table role="presentation" border="0" cellpadding="0" cellspacing="0"
-     style="width: {#if action.context.advisor.size == 1}50{#else}100{/if}%; table-layout: fixed;">
+    {#let width100=(new_recs && resolved_recs)}{#let width100=(new_recs && deact_recs || width100)}{#let width100=(resolved_recs && deact_recs || width100)}
+    <table role="presentation" border="0" cellpadding="0" cellspacing="0" style="width: {#if width100}100{#else}50{/if}%; table-layout: fixed;">
+    {/let}{/let}{/let}
```

Comment:
Does this produce the output you need?
```suggestion
    <table role="presentation" border="0" cellpadding="0" cellspacing="0" style="width: {#if (new_recs && resolved_recs) || (new_recs && deact_recs) || (resolved_recs && deact_recs)}100{#else}50{/if}%; table-layout: fixed;">
```

Reply:
Yes, that does the trick!  Thanks @gwenneg !  Now why didn't I think of that :/